<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>805</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Splash</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="sideBar">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QListView" name="animationListView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="animationListControlContainer">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QToolButton" name="addAnimationButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>resource/add.png</normaloff>resource/add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="removeAnimationButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>resource/remove.png</normaloff>resource/remove.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="animationDataContainer">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="CanvasAndProperty">
          <item>
           <layout class="QVBoxLayout" name="KeyFrameDataBox">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QGroupBox" name="gridInfoBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>MS UI Gothic</family>
               </font>
              </property>
              <property name="title">
               <string>Info</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <widget class="QLabel" name="AnimationName">
                 <property name="text">
                  <string>Name</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEditAnimationName">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="paletBox">
              <property name="title">
               <string>Palets</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QGroupBox" name="imagePaletGroupBox">
                 <property name="title">
                  <string>Images</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <layout class="QHBoxLayout" name="PaletSelectBox_4">
                    <item>
                     <widget class="QToolButton" name="showPaletButton_4">
                      <property name="text">
                       <string>...</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>resource/folder_open.png</normaloff>
                        <normalon>resource/folder.png</normalon>
                        <selectedoff>resource/play_down.png</selectedoff>resource/folder_open.png</iconset>
                      </property>
                      <property name="checkable">
                       <bool>false</bool>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                      <property name="autoExclusive">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="PaletSelectBox_3">
                    <item>
                     <widget class="QToolButton" name="showPaletButton_3">
                      <property name="text">
                       <string>...</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>resource/folder_open.png</normaloff>
                        <normalon>resource/folder.png</normalon>
                        <selectedoff>resource/play_down.png</selectedoff>resource/folder_open.png</iconset>
                      </property>
                      <property name="checkable">
                       <bool>false</bool>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                      <property name="autoExclusive">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="PaletSelectBox_2">
                    <item>
                     <widget class="QToolButton" name="showPaletButton_2">
                      <property name="text">
                       <string>...</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>resource/folder_open.png</normaloff>
                        <normalon>resource/folder.png</normalon>
                        <selectedoff>resource/play_down.png</selectedoff>resource/folder_open.png</iconset>
                      </property>
                      <property name="checkable">
                       <bool>false</bool>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                      <property name="autoExclusive">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="PaletSelectBox_1">
                    <item>
                     <widget class="QToolButton" name="showPaletButton_1">
                      <property name="text">
                       <string>...</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>resource/folder_open.png</normaloff>
                        <normalon>resource/folder.png</normalon>
                        <selectedoff>resource/play_down.png</selectedoff>resource/folder_open.png</iconset>
                      </property>
                      <property name="checkable">
                       <bool>false</bool>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                      <property name="autoExclusive">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="horizontalGroupBox">
                 <property name="title">
                  <string>Animation</string>
                 </property>
                 <layout class="QHBoxLayout" name="animationPaletBox">
                  <property name="leftMargin">
                   <number>1</number>
                  </property>
                  <property name="rightMargin">
                   <number>1</number>
                  </property>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_8">
                    <item>
                     <widget class="QComboBox" name="animationComboBox"/>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QToolButton" name="showSubAnimationButton">
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset>
                      <normaloff>resource/play.png</normaloff>
                      <normalon>resource/play_down.png</normalon>resource/play.png</iconset>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="keyFramesBox">
              <property name="title">
               <string>Key Frames</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QTableWidget" name="keyFramesTableWidget">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>150</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="verticalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOn</enum>
                 </property>
                 <property name="horizontalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                 </property>
                 <property name="selectionMode">
                  <enum>QAbstractItemView::ExtendedSelection</enum>
                 </property>
                 <property name="selectionBehavior">
                  <enum>QAbstractItemView::SelectRows</enum>
                 </property>
                 <property name="cornerButtonEnabled">
                  <bool>true</bool>
                 </property>
                 <property name="rowCount">
                  <number>0</number>
                 </property>
                 <property name="columnCount">
                  <number>2</number>
                 </property>
                 <attribute name="horizontalHeaderVisible">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="horizontalHeaderCascadingSectionResizes">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>40</number>
                 </attribute>
                 <attribute name="horizontalHeaderHighlightSections">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="horizontalHeaderMinimumSectionSize">
                  <number>30</number>
                 </attribute>
                 <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="horizontalHeaderStretchLastSection">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="verticalHeaderVisible">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="verticalHeaderDefaultSectionSize">
                  <number>20</number>
                 </attribute>
                 <attribute name="verticalHeaderHighlightSections">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="verticalHeaderMinimumSectionSize">
                  <number>12</number>
                 </attribute>
                 <attribute name="verticalHeaderStretchLastSection">
                  <bool>false</bool>
                 </attribute>
                 <column>
                  <property name="text">
                   <string>Time</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Comment</string>
                  </property>
                 </column>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="keyFramesControls">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="addKeyFrameButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Add</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="insertKeyFrameButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>30</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>40</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Insert</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="removeKeyFrameButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Remove</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="eventGroupBox">
              <property name="title">
               <string>Events</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QTableWidget" name="eventTableWidget">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>150</height>
                  </size>
                 </property>
                 <property name="verticalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOn</enum>
                 </property>
                 <property name="horizontalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                 </property>
                 <property name="selectionBehavior">
                  <enum>QAbstractItemView::SelectRows</enum>
                 </property>
                 <attribute name="horizontalHeaderVisible">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>40</number>
                 </attribute>
                 <attribute name="horizontalHeaderHighlightSections">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="horizontalHeaderStretchLastSection">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="verticalHeaderVisible">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="verticalHeaderDefaultSectionSize">
                  <number>20</number>
                 </attribute>
                 <attribute name="verticalHeaderHighlightSections">
                  <bool>false</bool>
                 </attribute>
                 <column>
                  <property name="text">
                   <string>No</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Type</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Value</string>
                  </property>
                 </column>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QPushButton" name="addEventButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Add</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="insertEventButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>30</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Insert</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="removeEventButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Remove</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="animationViewer"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset>
     <normaloff>resource/folder_open.png</normaloff>resource/folder_open.png</iconset>
   </property>
   <property name="text">
    <string>OpenFile</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="icon">
    <iconset>
     <normaloff>resource/new.png</normaloff>resource/new.png</iconset>
   </property>
   <property name="text">
    <string>NewProject</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
